ingestors/jsonschema.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python3
+
+from typing import NamedTuple
+
+"""
+This module provides a flat Location structure so scraper authors do not need
+to understand the normalized ndjson spec:
+
+https://docs.google.com/document/d/1qxABDlRep76llrXhgwdwA6Ji3Jub-MftRvustk12XHc
+"""
+
+
+class Location(NamedTuple):
```

Comment:
This looks like a schema defintion for a `VaccineSpotter` location, perhaps we should call it `VaccineSpotterLocation`?